### ISSUE TITLE
Cluster ordering on multi-clusters machines

### ISSUE THREAD

**Issue by kmoppel** [zalando/pg_view#12]
Currently seems there is no ordering, but would be nice to have it:)

---

 rest93 9.3 master database connections: 110 of 800 allocated,                1 active
 type dev    fill   total    left   read  write    await path_size path
 data dm-0    0.0   2.1TB 239.1GB   10.4    1.0   6897.1    47.8GB /data1/postgres/pgsql_rest93/9.3/data
 xlog sda9    0.0 119.4GB  71.9GB    0.0    0.3      6.2     4.0GB /data1/postgres/pgsql_rest93/9.3/data/pg_xlog
   pid type    s utime stime guest read write   age db             user         query
...

 rest91 9.3 master database connections: 277 of 800 allocated,                2 active
 type dev    fill   total    left   read  write    await path_size path
 data dm-0    0.0   2.1TB 239.1GB    9.3    1.0   5841.3   343.7GB /data1/postgres/pgsql_rest91/9.3/data
 xlog sda9    0.0 119.4GB  71.9GB    0.0    0.3      4.0     4.0GB /data1/postgres/pgsql_rest91/9.3/data/pg_xlog
   pid type       s utime stime guest read write   age db             user         query
...

 rest92 9.3 master database connections: 949 of 1200 allocated,                17 active
 type dev    fill   total    left   read  write    await path_size path
 data dm-0   -2.5   2.1TB 239.1GB    8.9    1.0   5340.4   641.1GB /data1/postgres/pgsql_rest92/9.3/data
 xlog sda9    0.0 119.4GB  71.9GB    0.0    0.3      4.0     4.0GB /data1/postgres/pgsql_rest92/9.3/data/pg_xlog
...

**Comment by LappleApple** [comment]
Hi--this issue dates back to Oct 30, 2014. Was it resolved? Can we close it?

**Comment by alexeyklyukin** [comment]
Not yet

**Comment by LappleApple** [comment]
@alexeyklyukin @kmoppel Hi! How about now? Or might we re-tag with "Help Wanted"?

**Comment by alexeyklyukin** [comment]
> On 01 Aug 2016, at 13:39, Lauri at Zalando <notifications@github.com> wrote:
> 
> @alexeyklyukin <https://github.com/alexeyklyukin> @kmoppel <https://github.com/kmoppel> Hi! How about now? Or might we re-tag with "Help Wanted”?
> 

Yes, help wanted would be good (for all pg_view requests). Otherwise, we would have to wait even longer.

Oleksii.
